### ISSUE TITLE
vim.error: Cannot make changes, 'modifiable' is off

### ISSUE THREAD

**Issue by phongvcao** [python-mode/python-mode#543]
I recently got this error when opening :PymodeDoc in Vim:

Error detected while processing function pymode#doc#show:
line    7:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/phongvcao/.vim/bundle/python-mode/pymode/**init**.py", line 37, in get_documentation
    vim.current.buffer.append(str(out).splitlines(), 0)
vim.error: Vim:E21: Cannot make changes, 'modifiable' is off

I am currently using Vim with +python3 support. When I went to the autoload/pymode/doc.vim file and commented out the 'setlocal nomodifiable' option, everything works again. It seems that was where the error was and I had to add the following lines to my vimrc:

autocmd FileType qf,rst :setlocal modifiable
autocmd FileType qf,rst :set modifiable

So basically I have resolved the error for myself but I just want to let you guys know the issue and later roll out a fix for it...
##
